conf/application.conf

```diff
@@ -117,6 +117,12 @@ microservice {
       host = localhost
       port = 8470
     }
+
+    service-configs {
+      host = localhost
+      port = 8460
+      cache.expiration = 10 minute
+    }
   }
```

Comment:
you'll need to raise a PR against app-config-base as well to set service-configs before you build